sample-spring-boot-echo/src/main/java/com/example/bot/spring/echo/EchoApplication.java

```diff
@@ -36,7 +39,7 @@ public static void main(String[] args) {
 
     @EventMapping
     public Message handleTextMessageEvent(MessageEvent<TextMessageContent> event) {
-        System.out.println("event: " + event);
+        log.info("event: " + event);
```

Comment:
Using pringln may be out of kindness for beginner who don't know a lombok easily understand `EchoApplication.`

But of cource it's bad manner... How about standard way with `LoggerFactory`.

Reply:
I agree. `LoggerFactory` should be used.

Reply:
Update confirmed. Thank you!